packages/templates/clients/websocket/test/README.md

```diff
@@ -0,0 +1,91 @@
+## Prerequisites
+
+Install [Podman](https://podman.io/docs/installation). 
+
+> Microcks needs some services running to simulate the infrastructure and this requires multiple resources. Docker is not the best solution for this, thus better use Podman that manages resources better.
+
+You can also install [docker-compose](https://docs.docker.com/compose/install/) that `podman compose` will later use.
+
+## Test Project
+
+This `test` directory contains acceptance tests that check different clients with tests written in their respective languages. So JavaScript client is tested with JavaScript test, and Python with Python tests, and so on.
+
+To run tests: `podman compose -f ./microcks-setup/microcks-podman.yml --profile tests up -d`
+
+> You need to remember about `--profile test` to run whole setup with tests. This way you ensure that proper importer container imports `__fixtures__/asyncapi-hoppscotch-server.yml` into Microcks and tests run against it.
+
+## Testing Clients with Microcks
+
+This instruction is just a set of notes about how to play locally with Microcks that we already use for automated testing.
+
+Tests also run in containers, although with provided configuration you can start also testing environment without running tests, just to play with Microcks and understand more how it works.
+
+### Concept
+
+Microcks is a tool for mocking. To test our generated clients, we need to mock the server. In other words, if we want to check if the generated client for Postman works well, we need to mock the server that the client will communicate with while testing.
+
+### Using Microcks locally
+
+> This instruction assumes you are located in directory where `microcks-setup` directory is located
+
+#### Start Microcks
+
+1. Start Microcks infrastructure: `podman compose -f ./microcks-setup/microcks-podman.yml up -d`.
+
+1. Check with `podman ps` command if all services are running. It may take few minutes to start all containers. You can also run special script that will confirm services are ready: `bash ./microcks-setup/checkMicrocksReady.sh`.
```

Comment:
```suggestion
1. Check with `podman ps` command if all services are running. It may take few minutes to start all containers. You can also run a special script that will confirm that services are ready: `bash ./microcks-setup/checkMicrocksReady.sh`.
```